opsdroid/web.py

```diff
@@ -13,8 +13,21 @@
 
 
 class Web:
-    """Create class for opsdroid Web server."""
+    """A Web class
 
+    This class will create opsdroid Web server
+
+    Args:
+        opsdroid: opsroid instance
+
+    Attributes:
+        opsdroid: opsroid instance
+        config: web config of opsdroid instance from configuration yaml files
+        web_app: web.Application instance containing web server configurations
+        runner: AppRunner of web_app
```

Comment:
This is an `aiohttp.web.AppRunner`. Please provide the full name to help people find more information.